projects/ad4110/zed/Makefile

```diff
@@ -0,0 +1,24 @@
+####################################################################################
+## Copyright (c) 2018 - 2021 Analog Devices, Inc.
+### SPDX short identifier: BSD-1-Clause
+## Auto-generated, do not modify!
+####################################################################################
+
+PROJECT_NAME := ad4110
```

Comment:
The project name is ad4110_zed in system_project.tcl

Reply:
fixed